deps.bzl

```diff
@@ -2214,8 +2134,8 @@ def prysm_deps():
     go_repository(
         name = "com_github_magiconair_properties",
         importpath = "github.com/magiconair/properties",
-        sum = "h1:ZC2Vc7/ZFkGmsVC9KvOjumD+G5lXy2RtTKyzRKO2BQ4=",
-        version = "v1.8.1",
+        sum = "h1:SesWF0c8l/IKQX0NlsED38qoBhUpneg5HIHNdy5LyEE=",
+        version = "v1.7.4-0.20170902060319-8d7837e64d3c",
```

Comment:
many of these dependency versions got reduced, I wonder why

Reply:
it's used with com_github_golang_gddo which hasn't changed so possibly a higher version was required for gateway